src/constants/constants.js

```diff
@@ -566,10 +566,19 @@ export const CONTRACT_EVENTS = {
 export const NODE_ENVIRONMENTS = {
     DEVELOPMENT: 'development',
     TEST: 'test',
+    DEVNET: 'devnet',
     TESTNET: 'testnet',
     MAINNET: 'mainnet',
 };
 
+export const CHAIN_IDS = {
```

Comment:
shouldn't there be different chain ids for each blockchain ?

Reply:
This is only used for migration between old and new uals it should not be used anywhere else, we should change the name of constant i guess so it's not confusing

Reply:
yes, or just define it in the migration file